vendor/licenses/grammar/vscode-scala.txt

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Did you create this file manually??

Reply:
Yes. I guess I missed some script that copies it. Is that right?

Reply:
Yes, the `script/add-grammar` can take care of everything for you when changing a grammar (see [the contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-source-of-a-syntax-highlighting-grammar)). Here, I think Travis CI is only complaining because of the filename that doesn't match the repository name. It's probably easier to fix that manually :-)

Reply:
Perfect! We only need to wait for a second validation by someone from GitHub staff now.